### ISSUE TITLE
Start researching accessibility standards for SkyUx

### ISSUE THREAD

**Issue by Blackbaud-PatrickOFriel** [blackbaud/skyux1#400]
Look into the items in the intro in the accessibility outline below:

https://blackbaudinc.app.box.com/notes/56673317054?s=oghr7g307m5akx05manwwwluree0u3bu

**Comment by Blackbaud-PatrickOFriel** [comment]
http://www.deque.com/products/aXe/

https://github.com/springernature/pa11y

https://addyosmani.com/a11y/

These three tools mentioned in the above document can be installed through npm and run through the command line so they look to be promising in terms of easily integrating with our current build process.
